Makefile

```diff
@@ -197,41 +226,63 @@ check_cert_manager:
                fi;\
        fi
 
+# Location to install dependencies
+LOCALBIN ?= $(shell pwd)/bin
+.PHONY: local-bin
+local-bin:
+	mkdir -p $(LOCALBIN)
+
+# Location to install test dependencies
+TESTBIN ?= $(shell pwd)/testbin
+.PHONY: test-bin
+test-bin:
+	mkdir -p $(TESTBIN)
+
 # Download controller-gen locally if necessary
-CONTROLLER_GEN = $(shell pwd)/bin/controller-gen
-controller-gen:
-	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.1)
+CONTROLLER_GEN = $(LOCALBIN)/controller-gen
+.PHONY: controller-gen
+controller-gen: local-bin
+	[ -f $(CONTROLLER_GEN) ] || GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-tools/cmd/controller-gen@v$(CONTROLLER_GEN_VERSION)
 
 # Download kustomize locally if necessary
-KUSTOMIZE = $(shell pwd)/bin/kustomize
-kustomize:
-	$(call go-get-tool,$(KUSTOMIZE),sigs.k8s.io/kustomize/kustomize/v3@v$(KUSTOMIZE_VERSION))
+KUSTOMIZE_INSTALL_SCRIPT ?= "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"
+KUSTOMIZE = $(LOCALBIN)/kustomize
+.PHONY: kustomize
+kustomize: local-bin
```

Comment:
I noticed in this guide, they use the path as the target (e.g. `$(KUSTOMIZE)` instead of `kustomize`): https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.21.0/#for-golang-based-operators--change-makefile-targets-to-replace-go-get-with-go-install

Is that something we can do, or did you run into a problem with it?

Reply:
Oh no there is no issue at all. I was following our style for targets that fetch tools. Let me follow their current way then!